### PR TITLE
feat:wishlist

### PR DESCRIPTION
**Pull request by ultronjs**
Adding two new page -
-wishList
-emptyWishList

### CODE REVIEW

**Review by Prabhav-Sharma — APPROVED** [review]

File: pages/wishListEmpty.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                </span>
+            </a>
+        </div>
+
+    </nav>
+    <div class="main_container flex flex-col flex-center py-small">
+        <div class="h3">YOUR WISHLIST IS EMPTY</div>
+        <div class="h5 color_grey_200">Add items that you like to your wishlist. Review them anytime and easily move
+            them to the bag.</div>
+        <img class="empty_image" src="../assets/empty_wishlist.png" alt="EMPTY">
```
Comment (Kothalkarpriya) [inline pages/wishListEmpty.html]: Should Practice having / before closing img tag

**Review by Kothalkarpriya — APPROVED** [review]
Good work, just a suggestion added

File: pages/wishListEmpty.html (hunk 1 above)
Comment (ultronjs) [reply pages/wishListEmpty.html]: Thanks for letting me know.Fixed it.

**Comment by ultronjs** [comment]
Merging after 2 approval and resolving the git comments

**Pull request merged**